app/src/main/java/com/example/music_app/data/repositories/playlists/ToPlaylistList.kt

```diff
@@ -0,0 +1,14 @@
+package com.example.music_app.data.repositories.playlists
+
+import com.example.music_app.data.models.Playlist
+import com.example.music_app.data.models.PlaylistsResponse
+
+fun PlaylistsResponse.toPlaylistList(): List<Playlist>? {
+    return items?.map { playlistItem ->
+        Playlist(
+            name = playlistItem.name,
+            description = playlistItem.description,
+            images = playlistItem.images
+        )
+    }
+}
```

Comment:
It would be better to create mappers for individual models like
```suggestion
fun PlaylistItem.toPlaylist() = Playlist(
            name = playlistItem.name,
            description = playlistItem.description,
            images = playlistItem.images
        )
```